.github/workflows/ci.yml

```diff
@@ -127,20 +127,6 @@ jobs:
           #
           curl -sL 'https://unofficial-builds.nodejs.org/download/release/v${{ matrix.node }}/node-v${{ matrix.node }}-linux-x64-glibc-217.tar.xz' | xzcat | sudo tar -vx  --strip-components=1 -C /usr/local/
           sudo npm install --global yarn
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
-
-      - uses: actions/cache@v3
-        id: yarn-cache
-        if: matrix.friendlyName == 'Ubuntu'
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key:
-            ${{ runner.os }}-${{ matrix.arch }}-yarn-${{
-            hashFiles('**/yarn.lock') }}
```

Comment:
hashFiles is a github actions expression. the ** is evaluating to `/home/runner/work/desktop/desktop` which is now where the yarn.lock file is located in the docker.

I think you should try passing in the directory `/__w/desktop/desktop` since I see that in other output of the CI

Reply:
Yeah, I've pulled that out for now to revisit later.